main.py

```diff
@@ -287,7 +286,13 @@ def log_bad_request(request):
 
 @app.route('/')
 def index(marker=None, message=None):
-    context = {'minimal_zoom': MINIMAL_ZOOM, 'url': request.base_url, 'index_url': request.url_root}
+    context = {'minimal_zoom': MINIMAL_ZOOM,
```

Comment:
I can define class that holds all constants in python. like this : [http://stackoverflow.com/a/23274028/61883](http://stackoverflow.com/a/23274028/61883)